helm-charts/Makefile

```diff
@@ -5,7 +5,8 @@ include ../Makefile.os
 RELEASE_VERSION ?= latest
 CHART_SEMANTIC_RELEASE_VERSION ?= $(shell cat ../release.version | tr A-Z a-z)
 CHART_NAME=strimzi-kafka-operator
-CHART_PATH=./$(CHART_NAME)/
+HELM_VERSION ?= helm3
```

Comment:
Probably will alleviate confusion if you doc as a comment above this that this is used both as the path to the specific chart as well as the name of the cli tool you are using to package/install it.

Now that I think on it more deeply I do wonder if we should split this out to:
```
HELM_CLI ?= helm
CHART_PATH ?= ./helm-charts/helm3/strimzi-kafka-operator/
```
And we override in travis HELM_CLI to be helm2/helm3

Reply:
I am slightly confused with the suggestion here... Do you mean to have something like the following?

```
HELM_CLI ?= helm
HELM_VERSION ?= 3
CHART_PATH ?= ./helm-charts/$(HELM_CLI)$(HELM_VERSION)/strimzi-kafka-operator/
```

What I am missing is how in this pipeline passes those env vars down to the Makefile. Are we talking about the 

```
  - TEST_HELM_VERSION=v2.16.3
  - TEST_HELM3_VERSION=v3.2.0
``` 

as well?

Reply:
I think `TEST_HELM_VERSION` -> should go to `TEST_HELM2_VERSION`

And I think the chart path should just be a single string that defaults to thje second chart, i.e. 
```
HELM_CLI ?= helm
CHART_PATH ?= ./helm-charts/helm2/strimzi-kafka-operator/
```
This means we can still build the helm3 chart by overriding the path and we can apply different paths to the cli for instance to build helm3 chart you could run 
```
HELM_CLI="/usr/bin/local/helm3" CHART_PATH="./helm-charts/helm3/strimzi-kafka-operator/" make ...
```

But maybe I am misunderstanding something here...